### ISSUE TITLE
BtsMeasureCardInflater.java: Only show currently supported Items

### ISSUE THREAD

**Issue by SecUpwN** [SecUpwN/Android-IMSI-Catcher-Detector#801]
>Thank you for your feedback! Please add below details so that we can help you a bit faster.

---

#### Agreements
>Please check these boxes to ensure you've actually read and understood these important rules!

- [x] **I have reviewed and accepted the [guidelines for contributing](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md) to this project.**
- [x] **I have searched the [open Issues](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/issues) and made sure I am not filing a duplicate.**
- [x] **I have read the notes on [debugging](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md#debugging) and will properly maintain this Issue.**

---

#### Overview
>Please add a short and easy to understand description of the Issue you've experienced here.

Cleaned up `BtsMeasureCardInflater.java` and moved `TODO` as well as `NOTE` into this Issue. 

 * TODO:
                
 - [ ] Order all the items according to appearance found in the DB table within `BtsMeasureCardInflater.java` (Is this the right way to do it?)

- [ ] Only show items currently supported
- [ ] Add record id "n/XXX"

* NOTE: LAC and CID are not directly in this table, but referenced
through the foreign key "bts_id" to the DBi_bts table.

---

#### References
>If your Issue is related to any other existing Issues or pull requests, please link them here.

#725

**Comment by f3ndot** [comment]
`BtsMeasureCardInflater.java` stores & displays RAT/Network Type values as the `TelephonyManager` integer constants vs the human-readable strings that are meaningful to us.

We should consider a refactor on how we go about displaying "LTE" to a user instead of "13" more than the hacky workaround I introduced in 210861e2b5b237b997f502c8cda13118b57564b9

**Comment by SecUpwN** [comment]
>We should consider a refactor on how we go about displaying "LTE" to a user instead of "13" more than the hacky workaround I introduced in 210861e2b5b237b997f502c8cda13118b57564b9

I agree. Users should be able to read that. @He3556, what do you propose for refactoring?
